src/pages/FirstUsePage.js

```diff
@@ -65,10 +67,9 @@ export class FirstUsePage extends React.Component {
   }
 
   get buttonText() {
-    const { syncState } = this.props;
     const { status } = this.state;
 
-    const { progressMessage, errorMessage, progress, total } = syncState;
+    const { progressMessage, errorMessage, progress, total } = this.props;
```

Comment:
because the root component isn't getting the sync state, have connected this component to redux to have it passed through